index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"> 
+    <title>Abel's Amazing Work</title>
+</head>
+<body>
+    <header id="header">
+        <nav>
+            <h3 id="welcome">Welcome</h3>
+            <svg xmlns="http://www.w3.org/2000/svg" class="icon-md" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+        </nav>
+    </header>
+    <main id="landing">
+        <div>
+            <h1>Hey there. I'm Abel<br><span class="greeny">I'm a software developer</span></h1>
+            <p id="main-support-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div id="logo-cloud">
+            <a href="#"><img src="./assets/svg/twitter.svg" alt="twitter"></a>
+            <a href="#"><img src="./assets/svg/Linkedin-icon.svg" alt="LinkedIn"></a>
+            <a href="#"><img src="./assets/svg/medium.svg" alt="Medium"></a>
+            <a href="#"><img src="./assets/svg/github.svg" alt="Github" ></a>
+            <a href="#"><img src="./assets/svg/angellist.svg" alt="Angellist"></a>
+        </div>
```

Comment:
- [ ] Kindly let's group these social media links together inside of a list element like the unordered list `<ul>` since this is actually a list of links grouped together, you may have something like this: 

```
<ul>
    <li>
          <a href="#"><img src="./assets/svg/twitter.svg" alt="twitter"></a>
    </li>
    <li>
          <a href="#"><img src="./assets/svg/Linkedin-icon.svg" alt="LinkedIn"></a>
    </li>
</ul>
```
Let's fix this please 😉